README.md

```diff
@@ -10,49 +10,246 @@ It has only one function: generate the [Fibonacci sequence](https://en.wikipedia
+* The **public key** can be used by others to validate the signature of the artifacts, and should be published
+
+The `gpg` tool that can manage signatures for you is available from [their website](https://gnupg.org/download/index.html). You can also install it using package managers such as [Homebrew](https://brew.sh/):
+
+| `brew install gpg` |
+| :---- |
+
+Generate a key pair with the following command, and fill in the required details when prompted.
+
+| `gpg --full-generate-key` |
+| :---- |
+
+Choose the recommended defaults for the type of key to be created. You can leave these selections empty and press Enter to accept the default values.
+
+\> At the time of writing, this is `ECC (sign and encrypt)` with `Curve 25519`. Older versions of `gpg` might default to `RSA` with a `3072` bit key size.
+
+Next, you’ll be prompted to set the expiration of the key. If you choose to create a key that automatically expires after a set amount of time, you’ll need to [extend its validity](https://central.sonatype.org/publish/requirements/gpg/#dealing-with-expired-keys) when it expires.
+
+You will be asked for your real name, email, and a comment. You can leave the comment empty.
+
+| `Please select what kind of key you want:`<br>`(1) RSA and RSA`  <br> `(2) DSA and Elgamal`  <br> `(3) DSA (sign only)`  <br> `(4) RSA (sign only)`  <br> `(9) ECC (sign and encrypt) *default*`  <br> `(10) ECC (sign only)   `  <br> `(14) Existing key from card`  <br> ` Your selection? 9`  <br><br> `Please select which elliptic curve you want:`  <br> `(1) Curve 25519 *default*`  <br> `(4) NIST P-384`  <br> `(6) Brainpool P-256`  <br> `Your selection? 1 `  <br><br> `Please specify how long the key should be valid.`  <br> `0 = key does not expire`  <br> `<n>  = key expires in n days`  <br> `<n>w = key expires in n weeks`  <br> `<n>m = key expires in n months`  <br> `<n>y = key expires in n years`  <br> `Key is valid for? (0) 0`  <br> `Key does not expire at all`   <br><br> `Is this correct? (y/N) y `  <br> `GnuPG needs to construct a user ID to identify your key.`|
+|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Using a Markdown table here is strange, and is rendered peculiarly in IJ (although that's probably a bug on IJ's side).


<img width="441" alt="image" src="https://github.com/user-attachments/assets/50f075a6-62d9-4a33-b923-2ee13770565e">